__tests__/app.test.js

```diff
@@ -27,6 +28,15 @@ describe("Testing endpoints in the app.js file", () => {
         });
       });
   });
+  test("GET: 200 - request to /api returns a list of all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const endpoints = response.body.endpoints;
+        expect(typeof endpoints).toBe("object");
+      });
```

Comment:
You could verify that the response contains the expected endpoints by directly comparing it, like this:    `expect(body.endpoints).toEqual(endpoints) `and require your endpoints file in at the top 💡